qkrehdwns/likelion-assignment/src/main/java/likelion/likelionassignment/posttag/domain/PostTag.java

```diff
@@ -0,0 +1,35 @@
+package likelion.likelionassignment.posttag.domain;
+
+import jakarta.persistence.*;
+import likelion.likelionassignment.post.api.domain.Post;
+import likelion.likelionassignment.tag.domain.Tag;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@Table(name = "post_tag")
+public class PostTag {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "post_tag_id")
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "post_id")
```

Comment:
외래 키 필수 조건이 코드상 의도는 존재하지만, nullable = false가 빠져 있는 것 같아요!
데이터 정합성을 위해서는 항상 명시적으로 작성해주는게 좋아요 😊😊

```suggestion
    @JoinColumn(name = "post_id", nullable = false)
```

Reply:
확인했습니다!